runtime/runtime.go

```diff
@@ -312,6 +312,20 @@ func Metadata(_, _ int32) int64 {
 		Metadata()
 }
 
+//go:export Metadata_at_version
```

Comment:
Function name must be `Metadata_metadata_at_version`
https://spec.polkadot.network/chap-runtime-api#sect-rte-metadata-metadata-at-version

Reply:
Fixed